packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -93,6 +93,15 @@ export interface IFluidContainerEvents extends IEvent {
 	 * {@link IFluidContainer.dispose}), this will contain details about the error that caused it.
 	 */
 	(event: "disposed", listener: (error?: ICriticalContainerError) => void);
+
+	/**
+	 * Emitted when the {@link IFluidContainer} readonly flag is changed.
```

Comment:
Nit (to improve the generated API docs experience):
```suggestion
	 * Emitted when {@link IFluidContainer.readonly} is changed.
```